src/js/common/schemaform/FormApp.jsx

```diff
@@ -102,6 +124,27 @@ class FormApp extends React.Component {
     return message;
   }
 
+  redirectOrLoad(props) {
+    const firstPagePath = props.routes[props.routes.length - 1].pageList[0].path;
+
+    // If we're logged in and have a saved / pre-filled form, load that
+    if (props.isLoggedIn) {
+      const currentForm = props.formConfig.formId;
+      const isPrefill = props.prefillsAvailable.includes(currentForm);
+      if (props.savedForms.includes(currentForm) || isPrefill) {
```

Comment:
Ah, I was thinking of this incorrectly. If we're always redirecting to the last page for a saved/prefilled form, it matters less. I still think the intro is better, but it's up to @stacyhlinn.